packages/ui-components/src/components/description-list/description-list.tsx

```diff
@@ -0,0 +1,38 @@
+import { Component, Fragment, Host, Prop, h } from '@stencil/core';
+import { IDescriptionList, IDescriptionListItem } from './description-list.types';
+import { TEXT_TOOLTIP_CONFIG } from './description-list.config';
+import { ETooltipPosition } from '../../types';
+
+@Component({
+	tag: 'kv-description-list',
+	styleUrl: 'description-list.scss',
+	shadow: true
+})
+export class KvDescriptionList implements IDescriptionList {
+	/** @inheritdoc */
+	@Prop({ reflect: true }) items!: IDescriptionListItem[];
+
+	render() {
+		return (
+			<Host>
+				<div class="description-list-container">
+					{this.items?.map(({ title, description, tooltipInfo }) => (
+						<Fragment>
+							<div class="title">{title}</div>
+							<div class="description">
+								<kv-tooltip text={tooltipInfo?.text && !tooltipInfo.icon ? tooltipInfo.text : ''} options={TEXT_TOOLTIP_CONFIG}>
+									{description}
+								</kv-tooltip>
+								{tooltipInfo?.icon && (
```

Comment:
We can have a tooltip and a toggle tip in the same description item? I don't think so. 🤔

Reply:
In case we have both, should we consider having two different config properties like:
- tooltipInfo
- toggletipInfo 

This to avoid having properties with a name of one component being used in another, ex: "<kv-toggle-tip text={tooltipInfo.text}"

Reply:
> We can have a tooltip and a toggle tip in the same description item? I don't think so. 🤔

Only one or the other